subprojects/docs/src/snippets/testKit/junitQuickstart/groovy/build.gradle

```diff
@@ -10,7 +10,12 @@ dependencies {
 
 // tag::declare-junit-dependency[]
 dependencies {
-    testImplementation 'junit:junit:4.13'
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.6.0")
```

Comment:
consider using the latest `5.7.1`

Reply:
Thanks for the review. I'll look into this separately.

Reply:
Here's the follow-up PR: https://github.com/gradle/gradle/pull/16427